bin/crew

```diff
@@ -1294,6 +1311,13 @@ def resolve_dependencies
   abort "Package #{@pkg.name} is not compatible with your device architecture (#{ARCH}) :/".lightred unless @device[:compatible_packages].any? {|elem| elem[:name] == @pkg.name }
 
   @dependencies = []
+  if @pkg.gnome?
+    GNOME_DEPS.each do |dep|
```

Comment:
If there are some gnome specific dependencies missing in some gnome packages, we should add those dependencies to the package recipes but not modify the dependency list.

Reply:
The argument here is to just make sure the gnome deps needed for the gnome postinstall get added in case they aren't installed. These are logical dependencies, not necessarily build or runtime dependencies. (Though in this case some are runtime deps too.)

Is that a mistake?

Reply:
I agree with @supechicken here.  We should include the dependencies in the packages instead.  There may be an edge case where we don't want the extra dependencies.  Better to control at the package level.

Reply:
Sounds good. I removed that entirely.